ui/app/pages/send/send-content/send-gas-row/send-gas-row.scss

```diff
@@ -1,10 +1,11 @@
 .currency-display {
+  @include Paragraph;
```

Comment:
This class is unused as it's solely used in the `GasFeeDisplay` component, which itself is unused

Reply:
Update: [`GasFeeDisplay` may in fact be used](https://github.com/MetaMask/metamask-extension/pull/9074#discussion_r468881458).